packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -21,6 +21,15 @@ import useApplicationsMenu from '../../hooks/use-applications-menu';
 import FetchProjectExtensionsNavbar from './fetch-project-extensions-navbar.settings.graphql';
 import nonNullable from './non-nullable';
 
+enum ActionTypes {
+  setActiveItemIndex = 'setActiveItemIndex',
+  unsetActiveItemIndex = 'unsetActiveItemIndex',
+  setIsExpanderVisible = 'setIsExpanderVisible',
+  toggleIsMenuOpen = 'toggleIsMenuOpen',
+  setIsMenuOpenAndMakeExpanderVisible = 'setIsMenuOpenAndMakeExpanderVisible',
+  reset = 'reset',
+}
```

Comment:
Can we use a const enum here? I presume this is not exported. In essence it saves bundle size. 

More here: https://ncjamieson.com/dont-export-const-enums/

Reply:
that is a nice one! ❤️ 
![image](https://user-images.githubusercontent.com/462507/82116860-d8560680-976c-11ea-8bee-d346e8ed8f6f.png)

Reply:
Apologies we can‘t use it as we use babel unless we add a plugin for the transform. You‘d have to use as const. Maybe it was still a nice learning regarding TS itself ✌🏼